sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests/OpenTelemetryTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace OpenTelemetry.Exporter.AzureMonitor.IntegrationTests
+{
+    public class OpenTelemetryTests : IClassFixture<OpenTelemetryWebApplicationFactory<WebApp.Startup>>
+    {
+        private readonly OpenTelemetryWebApplicationFactory<WebApp.Startup> factory;
+
+        public OpenTelemetryTests(OpenTelemetryWebApplicationFactory<WebApp.Startup> factory)
+        {
+            this.factory = factory;
+        }
+
+        [SkippableFact]
+        public async Task ProofOfConcept()
+        {
+            string testValue = Guid.NewGuid().ToString();
+
+            // Arrange
+            var client = this.factory.CreateClient();
+            var request = new Uri(client.BaseAddress, $"api/home/{testValue}");
+
+            // Act
+            var response = await client.GetAsync(request);
+
+            // Assert
+            response.EnsureSuccessStatusCode();
+
+            // TODO: HOW TO REMOVE THE WAIT? CAN THE EXPORTER BE FLUSHED?
+            Task.Delay(5000).Wait();
+
+            /// TODO: I'M TRYING TO CALL SHUTDOWN TO GUARENTEE THAT THE EXPORTER IS FLUSHED WITHOUT NEEDING A WAIT.
```

Comment:
Refer to the spec:
* https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#shutdown
* https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#shutdown-1

Reply:
woohoo! looks like ForceFlush on the ActivityProcessor does the trick! :)